addons/survey/views/survey_templates_statistics.xml

```diff
@@ -217,13 +220,20 @@
                                 </t>
                             </td>
                             <td>
-                                <t t-if="question.question_type == 'text_box'">
-                                    <span t-if="no_print_url" t-field="input_line.value_text_box" />
-                                    <a t-else="" t-att-href="input_line.user_input_id.get_print_url()" t-field="input_line.value_text_box"></a>
+                                <!-- If answer already covered by filter or if there is only one line to display, do not allow filtering on it again -->
+                                <t t-if="question.id in disabledQuestionsFilters or len(table_data) == 1">
+                                    <span t-field="input_line.value_char_box" t-if="question.question_type == 'char_box'"/>
+                                    <span t-field="input_line.value_text_box" t-if="question.question_type == 'text_box'"/>
                                 </t>
-                                <t t-if="question.question_type == 'char_box'">
-                                    <span t-if="no_print_url" t-field="input_line.value_char_box" />
-                                    <a t-else="" t-att-href="input_line.user_input_id.get_print_url()" t-field="input_line.value_char_box"></a>
+                                <t t-else="">
+                                    <a class="text-primary filter-add-answer"
+                                        data-model-short-key="L" t-att-data-answer-id="input_line.id"
+                                        role="button" aria-label="Filter surveys" title="Only show survey results having selected this answer">
+                                        <t t-if="question.question_type in ('char_box', 'text_box')" t-esc="input_line['value_' + question.question_type]"/>
```

Comment:
2 things here:

- Aren't we already in "char_box" / "text_box" template? Making the test redundant.
- As said in other comments, would avoid using constructs to get the value, don't we have a magic new method now?